package.json

```diff
@@ -1,125 +1,221 @@
+	},
+	"license": "MIT",
+	"version": "0.12.1-dev.1",
+	"publisher": "sourcery",
+	"icon": "sourcery-icon.png",
+	"keywords": [
+		"Python",
+		"refactoring"
+	],
+	"engines": {
+		"vscode": "^1.65.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"activationEvents": [
+		"onLanguage:python",
+		"onStartupFinished"
+	],
+	"contributes": {
+		"walkthroughs": [
+			{
+				"id": "sourceryWelcome",
+				"title": "Get started with Sourcery",
+				"description": "Your first steps to try out the powerful tools and features that the Sourcery extension has to offer!",
+				"when": "workspacePlatform != webworker",
+				"steps": [
+					{
+						"id": "sourcery.openWelcomeFile",
+						"title": "Get your first recommendation",
+						"description": "Get started by opening our tutorial \n[Open tutorial](command:toSide:sourcery.welcome.open)\n Put the cursor over the underlined code to see the refactoring suggestion. \n You can see all Sourcery's recommendations in the [Problems Pane](command:workbench.actions.view.problems)",
+						"media": {
+							"markdown": "walkthrough/view_welcome_file.md"
+						},
+						"when": "",
+						"completionEvents": [
+							"onCommand:sourcery.welcome.open"
+						]
+					},
+					{
+						"id": "sourcery.acceptRecommendation",
+						"title": "Accept your first recommendation",
+						"description": "Put the cursor on an underline, then click the lightbulb or use the quick fix hotkey to show the code actions.\n Take the `Sourcery - Convert for loop into list comprehension` option to accept the change.\n",
+						"media": {
+							"markdown": "walkthrough/accept_recommendation.md"
+						},
+						"when": "",
+						"completionEvents": [
+							"onContext:acceptRecommendationContextKey"
+						]
+					},
+					{
+						"id": "sourcery.configuration",
+						"title": "Configure Sourcery to your needs.",
+						"description": "Create a configuration file in your project to customise Sourcery\n[Create config file](command:toSide:sourcery.config.create)\n ",
+						"media": {
+							"markdown": "walkthrough/configuration.md"
+						},
+						"completionEvents": [
+							"onCommand:sourcery.config.create"
+						]
+					},
+					{
+						"id": "sourcery.customRules",
+						"title": "Add custom rules",
+						"description": "You can add your own custom rules to Sourcery!",
+						"media": {
+							"markdown": "walkthrough/custom_rules_tutorial.md"
+						}
+					},
+					{
+						"id": "sourcery.enterToken",
+						"title": "Enter your token",
+						"description": "Sign up for a Sourcery token to get a free trial of Sourcery Pro for 30 days\n[Sign up for token](https://sourcery.ai/signup)\n[Enter token](command:toSide:workbench.action.openSettings?\"sourcery\")",
+						"media": {
+							"markdown": "walkthrough/enter_token.md"
+						},
+						"completionEvents": [
+							"onSettingChanged:sourcery.token"
+						]
+					},
+					{
+						"id": "sourcery.scanRefactorings",
```

Comment:
I'm not sure if this is worth including here.

An alternative final step is to tell people to browse our docs and include links to e.g:
- Docs main page
- Scan for refactorings
- Detect clones
- Config

Reply:
Fleshing this out in follow-up PR